### ISSUE TITLE
Accessibility: VoiceOver cannot expand/collapse folders when using Miniflux (Google Reader API) account

### ISSUE THREAD

**Issue by basyigit** [Ranchero-Software/NetNewsWire#5207]
### Environment * Device: iPhone * OS: iOS 17+ * App Version: Latest NetNewsWire * Account Type: Miniflux (Google Reader API)

**Comment by brentsimmons** [comment]
Fixed in 7.1.
